docs/index.asciidoc

```diff
@@ -87,6 +88,16 @@ filter {
   }
 }
 
+[id="plugins-{type}s-{plugin}-parser_options"]
+===== `parser_options`
+
+* Value type is <<string,string>>
+
+Setting XML parser options allows for more control of the parsing process.
+By default the parser is not strict and thus accepts some invalid content.
+Currently supported options are:
+
+ - _strict_ - forces the parser to fail early when content is not valid xml
```

Comment:
@karenzone could you please check these for me.
we might add more to the list later (unofficially other options work) but for now only strict is to be documented